.test-infra/metrics/influxdb/Dockerfile

```diff
@@ -0,0 +1,35 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+FROM python:3-slim
+
+RUN pip install --no-cache-dir gsutil
+
+WORKDIR /
+
+RUN gsutil cp gs://apache-beam-testing-metrics/influxdb-backup.tar.gz . && \
```

Comment:
Are these backups available to non-committers?

Reply:
Everyone on the Internet has read-only access to the bucket. Grafana dashboards are already public, so it doesn't change anything in terms of security or privacy